lib/modules/manager/kubernetes/__fixtures__/underscore-tag.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: apps/v1
```

Comment:
please inline to test with codeBlock template helper

Reply:
I think the others in this file are all Fixtures, so it's ok